src/test/WorldIDIdentityManager.t.sol

```diff
@@ -83,7 +90,9 @@ contract WorldIDIdentityManagerTest is Test {
     /// @dev It is run before every single iteration of a property-based fuzzing test.
     function setUp() public {
         verifier = new SimpleVerifier();
-        makeNewIdentityManager(initialRoot, verifier);
+        stateBridge = new SimpleStateBridge();
+        stateBridgeProxy = address(stateBridge);
```

Comment:
Please label this address as is done for the others on the below lines.

Reply:
done, will be in next commit

Reply:
done